### PR TITLE
feat: [CDS-37852]: Service Dashboard Active Instances phase 2

### PR DESCRIPTION
**Pull request by guycastel**
### Summary

Generated Active Service Instances tables in card and popup according to new API response.

#### Screenshots

After - observe the table in Active Service Instances card and its more details popup (different column order, added clusters and pipelines columns):
image
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by bhavya-sinha — DISMISSED** [review]

**Comment by guycastel** [comment]
retrigger titlecheck

**Comment by guycastel** [comment]
retrigger coverage

**Comment by guycastel** [comment]
retrigger jest

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    projectIdentifier,
+    serviceId
+  }
+
+  const { data: deploymentData } = useGetEnvArtifactDetailsByServiceId({
+    queryParams: queryParamsDeployments
+  })
+
+  /* istanbul ignore next */
+  const isDeploymentTab = (): boolean => {
```
Comment (udayvunnam) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx:62]: shouldn't this be testable?

File: src/modules/75-cd/components/ServiceDetails/__tests__/ActiveServiceInstancesContentV2.test.tsx [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+    const { getByText } = render(
+      <TestWrapper>
+        <ActiveServiceInstancesV2 />
+      </TestWrapper>
+    )
+
+    const moreDetailsButton = getByText('cd.serviceDashboard.moreDetails')
+    await act(async () => {
+      fireEvent.click(moreDetailsButton!)
+    })
```
Comment (udayvunnam) [inline src/modules/75-cd/components/ServiceDetails/__tests__/ActiveServiceInstancesContentV2.test.tsx:239]: can be replaced with userEvent.click() and we can avoid act

File: src/modules/75-cd/components/ServiceDetails/__tests__/ActiveServiceInstancesContentV2.test.tsx [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+    jest.spyOn(cdngServices, 'useGetActiveServiceInstances').mockImplementation(() => {
+      return { loading: false, error: false, data: mockData, refetch: jest.fn() } as any
+    })
+    const { container } = render(
+      <TestWrapper>
+        <ActiveServiceInstancesV2 />
+      </TestWrapper>
+    )
+
+    const deploymentsTab = container.querySelectorAll('div[role="tab"]')[1]
```
Comment (udayvunnam) [inline src/modules/75-cd/components/ServiceDetails/__tests__/ActiveServiceInstancesContentV2.test.tsx:317]: getByRole('tab')?

**Comment by guycastel** [comment]
retrigger jest

**Comment by guycastel** [comment]
retrigger typecheck

**Review by vkbansal — APPROVED** [review]

**Pull request merged**